README.md

```diff
@@ -2,11 +2,50 @@
 
 ## Stage 1
 
-This is a [proposal](https://tc39.github.io/process-document/) to introduce a separator in numeric literals in [ECMAScript](https://github.com/tc39/ecma262/).
+This is a [proposal](https://tc39.github.io/process-document/), the result of a merge between an earlier draft of itself and Christophe Porteneuve's [proposal-numeric-underscores](https://github.com/tdd/proposal-numeric-underscores), extends the existing [_NumericLiteral_](https://tc39.github.io/ecma262/#prod-NumericLiteral) to allow a separator character between digits.
+
 
 ## Motivation
 
-This feature enables developers to make their numeric literals more readable by creating a visual separation between groups of digits. 
+This feature enables developers to make their numeric literals more readable by creating a visual separation between groups of digits. Large numeric literals are difficult for the human eye to parse quickly, especially when there are long digit repetitions. This impairs both the ability to get the correct value / order of magnitude...
+
+```js
+1000000000   // Is this a billion? a hundred millions? Ten millions?
+101475938.38 // what scale is this? what power of 10?
+```
+
+...but also fails to convey some use-case information, such as fixed-point arithmetic using integers.  For instance, financial computations often work in 4- to 6-digit fixed-point arithmetics, but even storing amounts as cents is not immediately obvious without separators in literals:
+
+
+```js
+const FEE = 12300;
+// is this 12,300? Or 123, because it's in cents?
+
+const AMOUNT = 1234500;
+// is this 1,234,500? Or cents, hence 12,345? Or financial, 4-fixed 123.45?
+```
+
+
+Using underscores (`_`, U+005F) as separators helps improve readability for numeric literals, both integers and floating-point (and in JS, it's all floating-point anyway):
+
+
+```ruby
```

Comment:
I think it makes more sense if the motivation had an example that was JS specific rather than ruby. Can we convert that original example into JS code (it already is i guess? just syntax highligher needs to change?) while keeping the brevity of the example?